components/js/SlickGrid.ts

```diff
@@ -343,15 +379,16 @@ export class SlickGrid implements OnChanges, OnInit, OnDestroy, AfterViewInit {
 
     /* tslint:disable:member-ordering */
     private getColumnEditor = (column: any): any => {
+        if (this.isColumnEditable && !this.isColumnEditable(this.getColumnIndex(column))) {
+            return undefined;
+        }
+
         let columnId = column.id;
-        let isEditable = this.editableColumnIds && this.editableColumnIds.indexOf(columnId) !== -1;
         let isColumnLoading = this.columnsLoading && this.columnsLoading.indexOf(columnId) !== -1;
-        if (isEditable) {
-            return isColumnLoading
-                ? getDisabledEditorClass('')
```

Comment:
I'm concerned by this one - it's possible to have a disabled column and indeed we want to have disabled columns in general. Why is undefined better? Can we fix up?